lib/Entities/SegmentEntity.php

```diff
@@ -105,4 +105,8 @@ public function setDescription($description) {
   public function getDynamicFilters() {
     return $this->dynamicFilters;
   }
+
+  public function isDynamic(): bool {
```

Comment:
Why do you think it is better to use this function instead of directly comparing the type?

Reply:
It is not used anywhere. Will remove it.